subprojects/docs/src/docs/userguide/dep-man/01-core-dependency-management/viewing_debugging_dependencies.adoc

```diff
@@ -1,220 +1,196 @@
 [[viewing-debugging-dependencies]]
-= Viewing and debugging dependencies
-
-Gradle provides sufficient tooling to navigate large dependency graphs and mitigate situations that can lead to link:https://en.wikipedia.org/wiki/Dependency_hell[dependency hell].
-Users can choose to render the full graph of dependencies as well as identify the selection reason and origin for a dependency.
-The origin of a dependency can be a declared dependency in the build script or a transitive dependency in graph plus their corresponding configuration.
-Gradle offers both capabilities through visual representation via build scans and as command line tooling.
-
 [[sec:debugging-build-scans]]
-== Build scans
+= View and Debug Dependencies
 
-[NOTE]
-====
-If you do not know what link:https://scans.gradle.com/[build scan]s are, be sure to check them out!
-====
+Gradle provides tooling to navigate dependency graphs and mitigate link:https://en.wikipedia.org/wiki/Dependency_hell[dependency hell].
+Users can render the full graph of dependencies as well as identify the selection reason and origin for a dependency.
+Dependencies can originate through build script declared dependencies or transitive dependencies.
+You can visualize dependencies with:
 
-A build scan can visualize dependencies as a navigable, searchable tree.
-Additional context information can be rendered by clicking on a specific dependency in the graph.
-
-.Dependency tree in a build scan
-image::dependency-management-dependencies-report-build-scan.png[]
+- the built-in Gradle CLI `dependencies` task
+- the built-in Gradle CLI `dependencyInsight` task
+- link:https://scans.gradle.com/[build scans]
 
 [[sec:listing_dependencies]]
-== Listing dependencies in a project
+== List Project Dependencies
 
-Gradle can visualize the whole dependency tree for every <<declaring_dependencies.adoc#sec:what-are-dependency-configurations,configuration>> available in the project.
+Gradle provides the built-in `dependencies` task to render a dependency tree from the command line.
+By default, the dependency tree renders dependencies for all <<declaring_dependencies.adoc#sec:what-are-dependency-configurations,configurations>>, but only a <<command_line_interface#executing_tasks_in_multi_project_builds,single project>>.
+The dependency tree indicates the selected version of each dependency.
+It also displays information about dependency conflict resolution.
 
-Rendering the dependency tree is particularly useful if you'd like to identify which dependencies have been resolved at runtime.
```

Comment:
I think this section should emphasize the distinction between the declared dependencies in a build file and the transitive deps that are resolved, which the report makes visible and are almost always the source of problems.